h5p-bildetema/src/components/PrintWords/PrintWords.tsx

```diff
@@ -98,16 +49,20 @@ export const PrintWords = forwardRef<HTMLDivElement, PrintWordsProps>(
       return chunksOfWords.map((chunk, index) => (
         <tr key={chunk.at(0)?.id} className={styles.tableRow}>
           {chunk.map(word => {
-            const wordLabel = extractWordLabel(word, showArticles);
+            // const wordLabel = extractWordLabel(word, showArticles);
+            const wordLabel = toSingleLabel(
+              word.translations.get(currentLanguageCode)?.labels || [],
+              showArticles,
+            );
 
             // TODO: Change method to find correct image from Swiper
-            const activeImage = word.images?.find(image => {
-              return document.querySelector(
-                `div.swiper-slide-active img[src="${image?.src}"]`,
-              );
-            });
+            // const activeImage = word.images?.find(image => {
+            //   return document.querySelector(
+            //     `div.swiper-slide-active img[src="${image?.src}"]`,
+            //   );
+            // });
```

Comment:
Why is this removed? Didn't we use it to print the correct image that is shown, so that users can select the preferred image to print?

Reply:
Fixed